README.md

```diff
@@ -23,6 +23,53 @@ providing an insightful and convenient interface to the Open Source project data
 
 ---
 
+## Contributing
+
+Please see our guide to contributing to this project at the following site: [CONTRIBUTORS.MD](https://github.com/oss-aspen/8Knot/blob/dev/docs/CONTRIBUTORS.md)
```

Comment:
What about the other file in docs? Do you think its still relivent?

Reply:
good point, I'll point to that too

Reply:
though we'll need to rewrite that asap